packages/kotti-ui/source/kotti-field-inline-edit/hooks.ts

```diff
@@ -0,0 +1,33 @@
+import { Ref, computed, watch, watchEffect } from '@vue/composition-api'
+
+import { KottiField } from '../kotti-field/types'
+
+import { KottiFieldInlineEdit } from './types'
+import { resizeTextArea } from './utils'
+
+export const useAdjustHeight = ({
+	field,
+	inputRef,
+	props,
+}: {
+	field: KottiField.Hook.Returns<KottiFieldInlineEdit.Value>
+	inputRef: Ref<HTMLTextAreaElement | HTMLInputElement | null>
+	props: KottiFieldInlineEdit.PropsInternal
+}) => {
+	if (!props.isMultiline) return
+	// Typecast needed because the type is logically implied from props.isMultiline
+	const _inputRef = inputRef as Ref<HTMLTextAreaElement | null>
+	const currentValue = computed(() => field.currentValue)
+
+	watchEffect(() => {
```

Comment:
maybe `watch` is better to avoid the `unused-vars`,
i did this:
```
watch([textareaRef, value], () => resizeTextarea(textareaRef.value), {
		immediate: true,
	})
```